wgsl/index.bs

```diff
@@ -8769,6 +8692,32 @@ WebGPU provides no guarantees about:
     the invocations of another workgroup.
     That is, you cannot assume that workgroups are launched in a particular order.
 
+## Fragment Shaders and Helper Invocations ## {#fragment-shaders-helper-invocations}
+
+Invocations in the [=fragment shader stage=] are divided into 2x2 grids of neighbouring invocations.
+Each of these grids is referred to as a <dfn noexport>quad</dfn>.
+Quads can collaborate in some collective operations (see [[#derivatives]]).
+
+An implementation will always ensure that a shader is executed with enough
+invocations such that each fragment is part of a full quad.
+When there are not enough fragments to fill the quad, an implementation will
+include <dfn noexport>helper invocations</dfn> for the missing members.
+
+[=write access|Write accesses=] performed by helper invocations in the
```

Comment:
Editorially, I'd probably preface this with the intent:

> Helper invocations are not observable, except for the fact they help compute [derivatives](#derivatives).  Therefore: ...


I think that's right?  (When subgroups arrive, things get murkier...)

Reply:
make that "except that they helper compute..."

Reply:
Also, I suggest making this a bulleted list